tasks/task865_bprec_classification.json

```diff
@@ -8,7 +8,7 @@
     "Categories": [
         "Classification"
     ],
-    "Definition": "In this task, you are given a polish text. Your task is to find the domain (banking, cosmetics, tele, electro) of the text.",
+    "Definition": "In this task, you are given a polish text about a product (for example a consumer comment on the social media) . Your task is to find the domain - banking, cosmetics, tele and electro of the text. The text contains products of 4 domains.",
```

Comment:
I think the definition still can improve. For example, you can explain the meaning of tele, electro, cosmetics, and banking.
Please use the appropriate encoding (ensure_ascii=False) to make JSON human-readable.